arch/cortex-m0/src/lib.rs

```diff
@@ -115,6 +115,7 @@ to_kernel:
   ldr r1, EXC_RETURN_MSP
   bx r1
 
+.align 2
```

Comment:
```suggestion
.align 4
```

Shouldn't this be word-aligned?  M0's don't support any amount of unaligned access, and that's being read by an `ldr`

Reply:
That's correct. Also corrected other instances of `.align 2` to `.align 4`.